ticket/src/main/resources/application.yml

```
@@ -0,0 +1,7 @@
+spring:
+  application:
+    name: ticket
+
+logging:
+  level:
+    root: WARN
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)